pkg/cmd/artefactor.go

```diff
@@ -98,6 +107,20 @@ func defaultValue(flagName string, defaultValue string) string {
 	return defaultValue
 }
 
+func getCredsFromFlags(c *cobra.Command) *util.Creds {
+	username := c.Flag(FlagDockerUserName).Value.String()
+	password := c.Flag(FlagDockerPassword).Value.String()
+
+	if len(username) > 0 {
```

Comment:
Is an empty password a valid case? Should we be checking it here also?

Reply:
As the auth for a registry is not our concern and can be handled by any solution (nginx / artefactory etc etc), we should allow it I think.